fluent/examples/apply-deploy.js

```diff
@@ -0,0 +1,22 @@
+//
+// Use this pattern to simulate kubectl apply -f; create a Deployment or replace it if it already exists.
+//
+const Client = require('kubernetes-client').Client
```

Comment:
We need to centralize this stuff on the `@kubernetes/node` client and config libraries.

Reply:
Yeah.  What order do you want to do that in? centralize before merging or after?

Reply:
I'm going to integrate the `@kubernetes/node` client+config lib with https://github.com/godaddy/kubernetes-client, release a new major version, and then update the PR.

Reply:
I'm making progress on this.  I'm deprecating the config and "backend" parts of `godaddy/kubernetes-client` and adding integration tests to help validate the transition to `@kubernetes/node`.

One issue: `@kubernetes/node` supports stream (or stream-like) interfaces for [watch endpoints](https://github.com/kubernetes-client/javascript/blob/master/src/watch.ts) and the [pods log endpoints](https://github.com/kubernetes-client/javascript/blob/master/src/log.ts).

`godaddy/kubernetes-client`, on the other hand, supports streaming for any endpoint's GET response.

Are there any plans or thinking around adding streaming to other endpoints in `@kubernetes/node`?  If not, I'll deprecate `godaddy/kubernetes-client`'s streaming APIs and add new ones that compliment the existing `@kubernetes/node` ones.

FWIW, I'm not sure what `godaddy/kubernetes-client` currently does is useful: I don't have any evidence that developers use streaming for anything other than logs and watch endpoints.

Reply:
I'm ok with adding streaming for other endpoints, but what is the use case? I don't think any other endpoints actually stream any data?

Reply:
I don't have any evidence that streaming for other endpoints is useful (I was wondering if you might).

At one point it was relatively common for Node.js code to use a Node.js stream to stream the result of a request (e.g., the results of `GET /api/v1/namespaces`).  I think that's largely fallen out of favor (or it has for me), especially with `async`/`await` support.

I'll move ahead with streaming support only for watch and log endpoints.

Reply:
What about pod exec (`/api/v1/namespaces/{namespace}/pods/{name}/exec`)?

Reply:
Oh, good point. I overlooked `exec` implementations because both clients return websockets.

`godaddy/kubernetes-client` will continue to return a websocket.